distribution.pm

```diff
@@ -282,12 +283,24 @@ sub script_output {
     }
 
     # and the markers including internal exit catcher
-    my $out = $output =~ /$marker(?<expected_output>.+)SCRIPT_FINISHED$marker-\d+-/s ? $+ : '';
-    # trim whitespaces
-    $out =~ s/^\s+|\s+$//g;
-    return $out;
+    my $out;
+    $out = $+{output} if ($output =~ /$marker(\r?\n|\025)(?<output>.*)SCRIPT_FINISHED$marker-$retval-/s);
+    return ($out, $retval);
+}
+
+=head2 script_output
+
+Wrap C<<script_output_with_status()>> and only return the
+script the trimmed output.
```

Comment:
that sentence reads weirdly now: "only returns the script the trimmed output"?

Reply:
> that sentence reads weirdly now: "only returns the script the trimmed output"?

[side note]
your use of an adverb makes it sound like the sentense is reading to you in a weird way. I fail to find if your english is correct or not though :)

Reply:
you mean it sounds to you like the sentence as an active object is reading something out to me? ;) https://forum.wordreference.com/threads/the-sentence-reads.2106906/#post-10541813

btw, there are double `<<` where should be single ones only, i.e. `Wrap C<script_output_with_status> and only …`

Reply:
Changed to 
```
Wrap C<script_output_with_status()> and only return the trimmed output.
Return empty string, if start- or end-markers where not found accordingly.
```

Reply:
s/where/were/

Reply:
thx for pointing that out!